rails/app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+	has_many :point_events
+	has_many :visits
+	has_many :reviews
+	has_many :establishments_visited, through: :visits, :source => :establishment
```

Comment:
I don't see why we need `establishments_visited` and `establishments_reviewed`. Waddupwidat?